documentation/modules/mtv-overview-page.adoc

```diff
@@ -10,7 +10,34 @@ The {project-first} *Overview* page displays system-wide information about migra
 
 If you have Administrator privileges, you can access the *Overview* page by clicking *Migration* -> *Overview* in the {ocp} web console.
 
-The *Overview* page displays the following information:
+The *Overview* page has 3 tabs:
+
+* Overview
+* YAML
+* Metrics
+
+[id="overview-tab_{context}"]
+== Overview tab
+
+The *Overview* tab lets you see:
+
+* Operator: The namespace on which the {project-short} Operator and the status of the Operator
+* Pods: The name, status, and creation time of each pod
+* Conditions: Status of the {project-short} Operator:
+
+** Failure: Last failure. `False` indicates no failure since deployment.
+** Running: Whether the Operator is currently running and waiting for the next reconciliation.
+** Successful: Last successful reconciliation.
+
+[id="yaml-tab_{context}"]
+== YAML tab
+
+The last manifest created or retrieved by the CLI is displayed. You can save, reload, or cancel your changes.
```

Comment:
```suggestion
The custom resource ForkliftController that defines the operation of the {project-short} Operator. You can modify the custom resource from this tab.
```
it's not very clear what manifest here means so I think it's better to say ForkliftController explicitly
in addition, saying "last" is problematic because it may not show the latest state (and then the user needs to press "reload".

@yaacov it's really nice to have this tab, I keep explaining on calls how to edit this CR from the CLI...